ocp_utilities/operators.py

```
@@ -67,23 +68,43 @@ def wait_for_install_plan_from_subscription(
     """
-    install_plan = wait_for_install_plan_from_subscription(admin_client=admin_client, subscription=subscription)
+    if timeout is not None:
+        warnings.warn(
+            "The 'timeout' parameter is deprecated and will be removed in a future release. "
+            "Please use 'creation_timeout' and 'status_complete_timeout' instead.",
+            DeprecationWarning,
+        )
+    else:
+        timeout = TIMEOUT_5MIN
+
+    creation_timeout = creation_timeout or timeout
```

Ruff

Undefined name `creation_timeout`

(F821)